.github/workflows/workflow.yaml

```diff
@@ -2,13 +2,14 @@ name: OpenGHG tests
 
 on:
   push:
-    branches: [master, devel]
+    branches:
+      - '*'
     tags:
       - "*"
     paths-ignore:
       - 'CHANGELOG.md'
   pull_request:
-    branches: [master, devel]
+    branches: [master, devel, new_object_store]
```

Comment:
Can `new_object_store` be removed here now?

Reply:
Yes please do remove it - this was put in a temporary fix for getting the CI to run (Prasad did add a note somewhere about this).